tests/transforms/general/testdata/custom_op_int_arg.mlir

```diff
@@ -0,0 +1,9 @@
+builtin.module {
+
+func.func @forward() {
+  %i = torch.constant.int 1000
+  torch.operator "torch.expand_custom_op_pass_test.int_arg"(%i) : (!torch.int) -> ()
```

Comment:
Should we use File check here?

Reply:
Good luck with that... We don't have it at this level, and any options to use it are... Not good

Reply:
That's sad... Tk could have also used filecheck style tests for fx tracing and codegen.

Reply:
We can try again. Last I tried was a few years ago.

Reply:
there is this python package that duplicates llvm filecheck: https://pypi.org/project/filecheck/ . But I don't know how okay it is to take a dependency like that. In theory, the package shouldn't ever need updates since the syntax is same.